config/v1/types_cluster_operator.go

```diff
@@ -176,10 +176,16 @@ const (
 	// unexpected errors are handled as operators mature.
 	OperatorDegraded ClusterStatusConditionType = "Degraded"
 
-	// Upgradeable indicates whether the operator safe to upgrade based on the current cluster state. When status is `False`
-	// administrators should not upgrade their cluster and the message field should contain a human readable description
-	// of what the administrator should do to allow the operator to successfully update.  A missing condition, True,
-	// and Unknown are all treated by the CVO as allowing an upgrade.
+	// Upgradeable indicates whether the operator is safe to upgrade based on the
+	// current cluster state. When status is False, the cluster-version operator
+	// will prevent the cluster from performing impacted updates unless forced.
+	// When set on ClusterVersion, the message will explain which updates are
```

Comment:
```suggestion
	// When set on ClusterVersion, the message will explain which updates(minor or patch) are
```

Reply:
this is my only nit, otherwise lgtm.

Reply:
Thanks; I've picked up that suggestion with 7ef0a538 -> 52ebfafc.